src/main/kotlin/org/rust/ide/folding/RustFoldingBuilder.kt

```diff
@@ -72,6 +72,15 @@ class RustFoldingBuilder() : FoldingBuilderEx(), DumbAware {
                 }
             }
 
+            override fun visitFnItem(o: RustFnItemElement) {
```

Comment:
Ah, am I correct that this is necessary, because visitor hierarchy changes after adding `RustPathNameElement` to `RustFnItem`?